react/utils/assets.ts

```diff
@@ -11,9 +11,9 @@ export const getVTEXImgHost = (account: string) => {
   return `https://${account}.vteximg.com.br`
 }
 
-const getAbsoluteURL = (account: string, url: string) => {
-  return isRelative(url)
-    ? `${getVTEXImgHost(account)}${url}`
+const getAbsoluteURL = (account: string, url: string, workspace: string, production: boolean) => {
+  return isRelative(url) && production
+    ? `${getVTEXImgHost(account)}${url}` + `?workspace=${workspace}`
```

Comment:
why not using the already existent string template like so: 
```suggestion
    ? `${getVTEXImgHost(account)}${url}?workspace=${workspace}`
```